site/content/en/docs/api_sdk/cli/_index.md

```diff
@@ -316,3 +321,25 @@ see that command's examples for more information.
   ```bash
   cvat-cli project ls --json > list_of_projects.json
   ```
+
+## Examples - functions
```

Comment:
I couldn't find a way to list possible parameters for the model (the `--function-parameter` parameter). It would be great to support this.

Reply:
It's not possible, because a function factory is just an arbitrary callable. The parameters could just be `**kwargs` (and for the torchvision functions, they _are_).

Reply:
I can suggest 2 options: 
- check function parameters and show if something is found (say n/a when unknown)
- provide API to declare the parameters

Anyway, it can be done later, just a future note.